<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>Reports</class>
  <widget class="QWidget" name="Reports">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>1200</width>
        <height>800</height>
      </rect>
    </property>
    <property name="styleSheet">
      <string notr="true">QWidget#Reports {
    background-color: #303030;
}
QLabel {
    color: white;
    font-size: 48pt;
}
QListWidget {
    background-color: #404040;
    color: white;
    border: 2px solid #505050;
    border-radius: 10px;
    padding: 10px;
    font-size: 16pt;
}
QListWidget::item {
    padding: 10px;
}
QListWidget::item:selected {
    background-color: #505050;
}
QPushButton {
    background-color: #404040;
    color: white;
    border: 2px solid #505050;
    border-radius: 10px;
    padding: 10px;
    font-size: 28pt;
    min-width: 180px;
    min-height: 70px;
}
QPushButton:hover {
    background-color: #505050;
    border-color: #606060;
}
QPushButton:pressed {
    background-color: #606060;
}</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
        <number>40</number>
      </property>
      <property name="leftMargin">
        <number>80</number>
      </property>
      <property name="topMargin">
        <number>80</number>
      </property>
      <property name="rightMargin">
        <number>80</number>
      </property>
      <property name="bottomMargin">
        <number>80</number>
      </property>
      <item>
        <widget class="QLabel" name="titleLabel">
          <property name="text"><string>Reports</string></property>
          <property name="alignment">
            <set>Qt::AlignCenter</set>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QListWidget" name="reportListWidget"/>
      </item>
      <item>
        <layout class="QHBoxLayout" name="buttonLayout">
          <item>
            <widget class="QPushButton" name="generateButton">
              <property name="text"><string>Generate Report</string></property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="viewButton">
              <property name="text"><string>View Report</string></property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="deleteButton">
              <property name="text"><string>Delete Report</string></property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="backButton">
              <property name="text"><string>Back</string></property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections/>
</ui>
